gulpfile.js

```diff
@@ -42,6 +43,8 @@ const CORDOVA_DIST_DIR = './dist_cordova/';
 
 const LINUX_INSTALL_DIR = '/opt/betaflight';
 
+const NODE_ENV = process.env.NODE_ENV || 'production';
```

Comment:
`NODE_ENV` is quite common for distinguishing prod and dev builds, but not used here. Might be worth introducing it?

Reply:
Which objective? To minify some things? To use different library to help debug? To me is ok if it helps to something.